src/blueapi/client/client.py

```diff
@@ -142,9 +143,19 @@ def get_task(self, task_id: str) -> TrackableTask[Task]:
         Returns:
             TrackableTask[Task]: Task details
         """
-
+        assert task_id != "", "Task ID cannot be empty"
```

Comment:
```suggestion
        assert task_id, "Task ID not provided!"
```

Because an empty string is False and None is also false, this catches a couple of exception cases